src/cli/build.rs

```diff
@@ -26,6 +30,67 @@ pub struct BuildCommand {
 impl Handler for BuildCommand {
     fn handler(&self) -> Result<()> {
         PreBuild::build_kaps("https://github.com/virt-do/kaps.git");
+        PreBuild::build_kernel();
         Ok(())
     }
 }
+
+pub struct PreBuild {}
+impl PreBuild {
+    /// When kaps will have its own release on GitHub, the binary could be fetched here.
+    /// Meanwhile, we are using git2 library to clone Kaps from GitHub, then cargo to build it from the sources.
+    /// If the kaps binary exists, skipping this step.
+    pub fn build_kaps(kaps_repository_url: &str) {
```

Comment:
Here as well, please return and propagate errors. We should fail gracefully, not panic.